src/page/template/modal/user-modal.htm

```diff
@@ -1,13 +1,22 @@
-<div data-bind="with: $root.userModal" >
+<div class="user-modal" data-bind="with: $root.userModal" >
   <modal params="isShown: isVisible, onBgClick: hide, onClosed: onClosed" data-bind="attr: {'data-uie-name': user() ? 'modal-user-profile' : ''}">
     <close-icon class="modal__close" data-bind="click: hide" data-uie-name="do-close"></close-icon>
     <!-- ko if: user() -->
       <panel-user-details params="participant: user"></panel-user-details>
-      <enriched-fields class="bottom-margin" params="userId: user().id"></enriched-fields>
-      <user-actions params="user: user(), actionsViewModel: actionsViewModel, onAction: onUserAction, isSelfActivated: userRepository.isActivatedAccount()"></user-actions>
+      <enriched-fields params="userId: user().id"></enriched-fields>
+      <user-actions params="user: user(), actionsViewModel: actionsViewModel, onAction: hide, isSelfActivated: userRepository.isActivatedAccount()"></user-actions>
     <!-- /ko -->
     <!-- ko ifnot: user() -->
```

Comment:
Reading those conditions it seems if the user is not found we will also display the loader